composer.json

```diff
@@ -17,37 +17,36 @@
 		"issues": "https://github.com/kdyby/aop/issues"
 	},
 	"require": {
-		"nette/di": "~2.4@dev",
-		"nette/php-generator": "~2.4@dev",
+		"nette/di": "~2.4",
+		"nette/php-generator": "~2.3",
 
-		"kdyby/annotations": "^2.1@dev",
-		"symfony/property-access": "~2.6"
+		"kdyby/annotations": "^2.1",
+		"symfony/property-access": "~3.2"
 	},
 	"require-dev": {
 		"php": ">=5.6",
-		"nette/application": "~2.4@dev",
-		"nette/bootstrap": "~2.4@dev",
-		"nette/caching": "~2.4@dev",
-		"nette/component-model": "~2.3@dev",
-		"nette/database": "~2.4@dev",
-		"nette/deprecated": "@dev",
-		"nette/di": "~2.4@dev",
-		"nette/finder": "~2.4@dev",
-		"nette/forms": "~2.4@dev",
-		"nette/http": "~2.4@dev",
-		"nette/mail": "~2.4@dev",
-		"nette/neon": "~2.4@dev",
-		"nette/php-generator": "~2.4@dev",
-		"nette/reflection": "~2.4@dev",
-		"nette/robot-loader": "~2.4@dev",
-		"nette/safe-stream": "~2.3@dev",
-		"nette/security": "~2.4@dev",
-		"nette/tokenizer": "~2.2@dev",
-		"nette/utils": "~2.4@dev",
-		"latte/latte": "~2.4@dev",
-		"tracy/tracy": "~2.4@dev",
+		"nette/application": "~2.4",
+		"nette/bootstrap": "~2.4",
+		"nette/caching": "~2.4",
+		"nette/component-model": "~2.3",
+		"nette/database": "~2.4",
+		"nette/di": "~2.4",
+		"nette/finder": "~2.4",
+		"nette/forms": "~2.4",
+		"nette/http": "~2.4",
+		"nette/mail": "~2.4",
+		"nette/neon": "~2.4",
+		"nette/php-generator": "2.6.2",
```

Comment:
I was playing with it and checking on which version will it fail, later i put it on newest while testing but forgot to remove it. In commit bellow this is removed.